Assets/VRTK/Scripts/VRTK_InteractGrab.cs

```diff
@@ -37,6 +37,8 @@ public class VRTK_InteractGrab : MonoBehaviour
         public float grabPrecognition = 0f;
         [Tooltip("An amount to multiply the velocity of any objects being thrown. This can be useful when scaling up the `[CameraRig]` to simulate being able to throw items further.")]
         public float throwMultiplier = 1f;
+        [Tooltip("Automatically scale the throw velocity up and down to match the scale of the `[CameraRig]`.")]
+        public bool throwScalesWithPlayer = true;
```

Comment:
Eh, it's up to you, but that seems a bit much. Even having the boolean at all is a compromise for me. I get that hypothetically you might want to set an exact max velocity. TransformDirection() used with the multiplier gives you that. In reality however, I've found that a percentage based approach gives much better results. TransformVector() with the multiplier gives you that option (e.g. throw at 70% power).

Is there a project that is using this? Cause you really really don't want to change it 😩